include/forum.php

```diff
@@ -0,0 +1,190 @@
+<?php
+
+/**
+ * @file include/forum.php
+ * @brief Functions related to forum functionality *
+ */
+
+/**
+ * @brief This class handles functions related to the forum functionality
+ */
+class forum {
```

Comment:
`Forums` would be enough. or `ForumManager`. That would leave `Forum ` class name for a future class that represent a single forum...
And file name should be same as class name (case included)